Milvus.Client/MilvusDataType.cs

```diff
@@ -77,4 +77,9 @@ public enum MilvusDataType
     /// FloatVector
     /// </summary>
     FloatVector = 101,
+
+    /// <summary>
+    /// Float16Vector
+    /// </summary>
+    Float16Vector = 102,
```

Comment:
Should we also introduce Float32Vector with the same value as FloatVector (101), and obsolete FloatVector, pointing to the new one? How does this look like on the Python side?

Reply:
Is it just for a consistent naming or is there a difference between float and float32?

Reply:
From https://milvus.io/api-reference/pymilvus/v2.6.x/MilvusClient/Collections/DataType.md#Constants

> FLOAT_VECTOR
> FLOAT16_VECTOR

Reply:
Just for consistent - and also clearer - naming.

Reply:
OK, the main thing IMHO is how it's documented in the Milvus docs; if they use `float` consistently to refer to float32 than that's fine by me.